src/playback.jl

```diff
@@ -0,0 +1,45 @@
+
+"""
+    RandomPolicy{RNG<:AbstractRNG, P<:Union{POMDP,MDP}, U<:Updater}
+a generic policy that uses the actions function to create a list of actions and then randomly samples an action from it.
+
+Constructor:
+
+    `PlaybackPolicy(actions::Vector{A}, backup_policy::Policy; logpdfs::Vector{Float64} = [])`
+
+# Fields
+- `actions::Vector{A}` a vector of actions to play back
+- `backup_policy::Policy` the policy to use when all prescribed actions have been taken but the episode continues
+- `logpdfs::Vector{Float64}` the log probability (density) of actions
+- `i::Int64` the current action index
+"""
+mutable struct PlaybackPolicy{A} <: Policy
+    actions::Vector{A}
+    backup_policy::Policy
+    logpdfs::Vector{Float64}
+    i::Int64
+end
+
+# Constructor for the PlaybackPolicy
+PlaybackPolicy(actions::Vector{A}, backup_policy::Policy; logpdfs::Vector{Float64} = Float64[]) where {A} = PlaybackPolicy(actions, backup_policy, logpdfs, 1)
```

Comment:
`A` is not used in the function so it might be removed.

Reply:
```
# Constructor for the PlaybackPolicy
PlaybackPolicy(actions::AbstractArray, backup_policy::Policy; logpdfs::AbstractArray{<:Real} = Float64[]) = PlaybackPolicy(actions, backup_policy, logpdfs, 1)
```

Reply:
Ok, removed A from the type